client/sdl/i_video_sdl20.cpp

```diff
@@ -712,6 +712,19 @@ bool ISDL20Window::isFocused() const
 	return mMouseFocus && mKeyboardFocus;
 }
 
+#if defined(SDL2016)
+
+//
+// ISDL20Window:flashWindow
+//
+// Flashes the taskbar icon if the window is not focused
+//
+void ISDL20Window::flashWindow() const {
+	if (!this->isFocused())
+		SDL_FlashWindow(mSDLWindow, SDL_FLASH_UNTIL_FOCUSED);
```

Comment:
I don't really like the idea of changing the shape of the class based on the SDL version. I think a better approach would be to move the `#ifdef` pair to be inside of the `ISDL20Window::flashWindow()` implementation.